.github/workflows/canary_release.yml

```diff
@@ -0,0 +1,24 @@
+name: 'Release Canary'
+on:
+  push:
+    branches:
+      - '4.x'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: yarn
+      - run: yarn install --ignore-scripts
+      - run: yarn prebuild
+      - run: yarn build:cjs      
```

Comment:
`- run: yarn build:types  ` should also call

Reply:
As you want to build everything it can be simplified
just add `-run: yarn` and remove these lines
```
 run: yarn install --ignore-scripts
      - run: yarn prebuild
      - run: yarn build:cjs     
      - run: yarn build:esm
      - run: yarn build:types
      - ```